test/test_tensorclass_nofuture.py

```diff
@@ -419,6 +434,35 @@ class MyDataMemMap3:
         data_wrong_cls[2:] = data1[2:]
 
 
+@pytest.mark.parametrize(
+    "val2broadcast",
+    [
+        0,
+        torch.zeros(4, 5),
+        TensorDict({"X": torch.zeros(2, 4, 5)}, batch_size=[2, 4]),
+        MemmapTensor.from_tensor(torch.zeros(4, 5)),
```

Comment:
Here we're creating objects from the library on the global space, which means that if memmap is broken we won't even launch the tests (since instantiation will occur at launch time, not during the test). Do you thing we can handle this differently?

Reply:
switched from object to types. It's ugly...I think there should be a more clean solution.